### ISSUE TITLE
cliphist no longer works after update of 2021-07-31

### ISSUE THREAD

**Issue by boudiccas** [redguardtoo/cliphist#18]
This is what shows in my config file, which until the update worked with parcellite -

````
#+begin_src emacs-lisp
(require 'cliphist)
(global-set-key (kbd "s-,") 'cliphist-select-item)
(global-set-key (kbd "s-.") 'cliphist-paste-item)
(setq cliphist-linux-clipboard-managers '("parcellite"))

(setq cliphist-select-item-callback
        (lambda (num str) (cliphist-copy-to-clipboard str)))
#+end_src
````

I was able to copy to the parcellite clipboard, and copy back from. 

Now i can't copy to parcellite from emacs, and neither copy back to emacs. So how can it be done please?

**Comment by boudiccas** [comment]
I've managed to get it working with the same configuration by placing the above code near the end of my config, so that's it read and activated twice. So how can I get it working with just one iteration please?

**Comment by redguardtoo** [comment]
Yes, check README. `cliphist-copy-to-clipboard `  is removed. There 3rd party packages (`xclip.el`, for example) which can copy the selected string into clipboard. I think cliphist should not re-invent the wheel.

**Comment by boudiccas** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Chen Bin ***@***.***> writes:

> Yes, check README. cliphist-copy-to-clipboard is removed. There 3rd party packages which can copy the selected string into clipboard. I think cliphist should not re-invent the wheel.
>
Well I'm sorry, but 'cliphist-copy-to-clipboard' is not shown in the
README.org file!

Please, if you're going to change something very fundamental in your
programme, please can you document somewhere a work-around so that
people can continue to use your programme?

You say that there are '3rd party packages', could you just list some of
them to help in using your programme?

Thanks
    Sharon.
- -- 
Debian 10.10, fluxbox 1.3.7, emacs 28.0.50, org 9.4.6
-----BEGIN PGP SIGNATURE-----

iQJPBAEBCgA5FiEELSc/6QwVBIYugJDbNoGAGQr4g1sFAmEID7EbHGJvdWRpY2Nh
c0Bza2ltYmxlLnBsdXMuY29tAAoJEDaBgBkK+INbMY0QAMQbXJ/+wJLliY1eXOvQ
6i+AAvoZidTQSyHycKNiz0m1wWzXhCXczsLLpAgtHvbaTO3UAYqKg4CAuPMpIk/1
9aftIe7/GpSp5s26mRq2ZmFi3PK2carbyPYeA9PhgZjcycioJuyyVcL190xaCodx
Ef1BqvWxE46RGBO+1qBrM5Ic7h9KHssOqoKeZ72q/D3dPeVTiJ8XX2Y1j5PSoZjT
JtKx5luNYxy1PGB7BsFy/UarhLS5rMl1A66pCQJOAs9OvfzCrmpi5i6I2rT7aAbO
Djg6li4WbBm33p6eMgAf2s0zRL9STSBJYomYz84EgLcBpvwdl8uq7XwwG/xSSc8n
ArVNQ9WO3Vm1VkAJzShftAXYPqYd8ukx5CPlXQOwO11H2iYLAa93LI3g8c+tx1a3
5kJr1y1wm0mF3Q7J73FsCTZGfuLHfX/iFfDp+QBMFcLGL9/pEE2DrZRTUcArWyym
6MUBXK90NEt0l1khcaW1pIMYL15hbNyPDrJeemWd2gKGAsaYnm+wpWukzj3TYbmh
2Bi+VY/KrTUK0Lt/RE2b8XK0tDUIFWPTTXkIu9jSRT2ofsjkdfQJZ494lqEnQ0X2
CbdjN0Ytbe53hXVFQ/AE0wINkU+8mG2vXKo553HIwCuBIEEKK9/rJWG0KGVbOQER
PZgyEzlgCTFacJ75sMVf1tNB
=bCM2
-----END PGP SIGNATURE-----

**Comment by redguardtoo** [comment]
caa2f10 cliphist-select-item-callback has default implementation (Chen Bin)
